### PR TITLE
clean up some instructions in README

### PR DESCRIPTION
**Pull request by cmarcelk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,7 @@ Install into a virtual environment (Python 3.6.1+).
 
     python3 -m venv venv
     . venv/bin/activate
-    pip3 install git+ssh://git@github.com/relaypro/relay-py.git#egg=relay-py
-    cd relay-py
-    pip3 install -e .
+    pip3 install git+https://git@github.com/relaypro/relay-py.git#egg=relay-py
```
Comment (steveims) [inline README.md:19]: I suggest using pip from the venv.  Upgrade it first.  So my steps are:

$ python3 -m venv venv
$ . venv/bin/activate
(venv)$ pip install -U pip
(venv)$ pip install git+https://git@github.com/relaypro/relay-py.git#egg=relay-py

File: README.md [hunk 2]
```diff
@@ -54,30 +52,26 @@ wf_server.start()
...
     . venv/bin/activate
     pip3 install -e .
 
-Start demo workflow server:
+Start demo workflow server (after setup):
 
-    bash
-    cd relay-py
-    . venv/bin/activate
     cd samples
```
Comment (steveims) [inline README.md:65]: is this run in the venv?

Comment (cmarcelk) [reply README.md:65]: I'll add a prompt to clarify.

File: README.md (hunk 1 above)
Comment (cmarcelk) [reply README.md:19]: Good idea. Will do.

**Review by steveims — APPROVED** [review]

**Pull request merged**